test/micro-distribution/lib/Standard/Base/0.0.0-dev/src/Main.enso

```diff
@@ -33,3 +33,6 @@ export project.Runtime
 
 import project.Warning
 export project.Warning
+
+allow_input_in env = @Builtin_Method "Runtime.allow_input_in"
+allow_output_in env = @Builtin_Method "Runtime.allow_output_in"
```

Comment:
And btw. shouldn't these definitions be placed also in the main `distribution` too? Otherwise, they are unreachable in the normal operations outside of the tests that replace the `distribution` with `micro-distribution`.

Because of that - I'd also suggest to add a copy of at least one tests from `IOContextTest.scala` to `test/Tests/src/Runtime/IO_Context_Spec.enso` so that we can ensure that they work outside of the micro-distribution framework properly (as I assume currently they wouldn't).

Or is that planned for some separate future PR?